index.html

```diff
@@ -39,13 +39,11 @@ <h2>I'am a Sotfware Developer</h2>
               </div>
           </div>
           <div class="social">
-            <ul>
-              <li><a href="#"><img src="images/github.png" alt="github"></a></li>
-              <li><a href="#"><img src="images/Linkedin%20icon.png" alt="linkin"></a></li>
-              <li><a href="#"><img src="images/angel%20list.png" alt="angel"></a></li>
-              <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
-              <li><a href="#"><img src="images/medium.png" alt="medium"></a></li>
-            </ul>
+              <a href="#"><img src="images/github.png" alt="github"></a>
+              <a href="#"><img src="images/Linkedin%20icon.png" alt="linkin"></a>
+              <a href="#"><img src="images/angel%20list.png" alt="angel"></a>
+              <a href="#"><img src="images/twitter.png" alt="twitter"></a>
+              <a href="#"><img src="images/medium.png" alt="medium"></a>
```

Comment:
It is better to use ul (unorder list) to <a>. I will organize in better way.

Reply:
Great job you almost did it :+1:  I think it would be better to use ul (unorder list), It is because your code will organize and look great.